modules/connected-to-disconnected-config-registry.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-config-registry_{context}"]
+= Configuring the cluster for the mirror registry
+
+After creating the mirror registry and mirroring the images to the registry, you need to make some modifications to your cluster in order for pods to be able to pull images from the mirror registry. 
+ 
+You must add the mirror registry credentials to the global pull secret, add the mirror registry server certificate to the cluster, and create an `ImageContentSourcePolicy` custom resource (ICSP), which associates the mirror registry with the source registry.
+
+. Add mirror registry credential to the cluster global pull-secret:
++
+[source,terminal]
+----
+$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=<pull_secret_location> <1>
+----
+<1> Provide the path to the new pull secret file.
++
+For example:
++
+[source,terminal]
+----
+$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=.mirrorsecretconfigjson
+----
+
+. Add the CA-signed mirror registry server certificate to the nodes in the cluster:
+
+.. Create a config map that includes the server certificate for the mirror registry
++
+[source,terminal]
+----
+$ oc create configmap <config_map_name> --from-file=<mirror_address_host>..<port>=$path/ca.crt -n openshift-config
+----
++
+For example:
++
+[source,terminal]
+----
+S oc create configmap registry-config --from-file=mirror.registry.com..443=/root/certs/ca-chain.cert.pem -n openshift-config
+----
+
+.. Use the config map to update the `image.config.openshift.io/cluster` custom resource (CR). {product-title} applies the changes to this CR to all nodes in the cluster:
++
+[source,terminal]
+----
+$ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"<config_map_name>"}}}' --type=merge
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"registry-config"}}}' --type=merge
+----
+
+. Create an ICSP to redirect container pull requests from the online registires to the mirror registry:
```

Comment:
s/registries/registires

Reply:
Sorry. Caught this one earlier. But forgot to save the file before committing.